src/user/user.controller.ts

```
@@ -113,25 +115,29 @@ export class UserController {
   @ApiHeader({
-    name: "academicyearid",
+    name: 'academicyearid',
   })
   async createUser(
     @Headers() headers,
     @Req() request: Request,
     @Body() userCreateDto: UserCreateDto,
     @Res() response: Response
   ) {
-    const academicYearId = headers["academicyearid"];
+    console.log('hii1');
```

ESLint

[error] Replace `'hii1'` with `"hii1"`

(prettier/prettier)